pages/linux/pdftohtml.md

```diff
@@ -0,0 +1,20 @@
+# pdftohtml
+
+> Pdftohtml is a program to convert PDF files into HTML, XML and PNG images.
+> More information: <https://linux.die.net/man/1/pdftohtml>.
```

Comment:
```suggestion
> Program to convert PDF files into HTML, XML and PNG images.
> More information: <https://manned.org/pdftohtml.1>.
```
We use manned as a fallback when there isn't an official documentation page.

Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links

Reply:
Oh, I read the style guide but didn't see that, sorry.